libs/ofxCv/src/Calibration.cpp

```diff
@@ -153,6 +153,105 @@ namespace ofxCv {
         updateUndistortion();
         ready = true;
     }
+    
+    void Calibration::loadLcp(string filename, float focalLength, bool absolute){
+        imagePoints.clear();
+        
+        // Load the XML
+        ofXml xml;
```

Comment:
This doesn't compile. I think it should be ofxXml, isn't it? @HalfdanJ @kylemcdonald